### PR TITLE
[PR] Preprod to Prod

### PR DESCRIPTION
**Pull request by lduf**
### Description
Fixing issues, improve cytoscape, improve duplicate, create view component, adding coordinate support, notification ,(invite user), GDPR support, refactoring & improve code, improve UX, import CSV

## Issues fixed
Ø

### Checklist
- [ ] Compiling
- [ ] Code factoring (comments, indent...)
- [ ] Documentation (net4p Doc, Readme...)

### Feature Checklist
- [ ] Cytoscape
- [ ] Person/Index
- [ ] Person/Filter
- [ ] Person/Create
- [ ] Person/Edit
- [ ] Person/Delete
- [ ] Relation/Index
- [ ] Relation/Filter
- [ ] Relation/Create
- [ ] Relation/Edit
- [ ] Relation/Delete
- [ ] Duplicates/Index
- [ ] Duplicates/Filter
- [ ] Duplicates/Create
- [ ] Duplicates/Edit
- [ ] ManageList/Delete
- [ ] ManageList/Create
- [ ] ManageList/Edit
- [ ] ManageField/Index
- [ ] ManageField/Create
- [ ] ManageField/Edit
- [ ] ManageField/Delete
- [ ] ApiList
- [ ] Crews
- [ ] Users

### CODE REVIEW

**Review by mathisfchx — APPROVED** [review]
🌍

File: src/tests/Feature/UserTest.php [hunk 1]
```diff
@@ -0,0 +1,626 @@
...
+        $response->assertStatus(403);
+    }
+
+    /* ------------------ requestRole ------------------ */
+
+    /**
+     * @brief An admin can request a new role IF : he isn't the only admin
+     */
+
+    public function test_user_with_permission_and_not_the_only_admin_can_request_role_from_the_profile_section() {
```
Comment (AntoinePiron) [inline src/tests/Feature/UserTest.php:85]: le nom de la fonction il est super long

Comment (lduf) [reply src/tests/Feature/UserTest.php:85]: true

Comment (mathisfchx) [reply src/tests/Feature/UserTest.php:85]: +1

**Pull request merged**